### PR TITLE
Django17 backwards compatible changes

### PR DESCRIPTION
**Pull request by emord**
should be all the backwards compatible changes from https://github.com/dimagi/commcare-hq/pull/7429

cc: @gcapalbo

### CODE REVIEW

File: corehq/apps/hqadmin/views.py [hunk 1]
```diff
@@ -410,8 +410,8 @@ def system_ajax(request):
                     traw['name'] = None
                 ret.append(traw)
             ret = sorted(ret, key=lambda x: x['succeeded'], reverse=True)
-            return HttpResponse(json.dumps(ret), mimetype = 'application/json')
-    return HttpResponse('{}', mimetype='application/json')
+            return HttpResponse(json.dumps(ret), content_type = 'application/json')
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: E251 unexpected spaces around keyword / parameter equals

**Comment by benrudolph** [comment]
:+1:

File: settings.py [hunk 2]
```diff
@@ -331,7 +331,6 @@
     'custom.world_vision',
     'custom.tdh',
     'custom.up_nrhm',
-    'crispy_forms',
```
Comment (gcapalbo) [inline settings.py]: Was removing crispy_forms intentional here? Just wanted to make sure because it seems unrelated to the rest of the PR

Comment (gcapalbo) [inline settings.py]: Never mind, I see it was duplicated
